src/model/transaction/LockHashAlgorithm.ts

```diff
@@ -23,21 +23,25 @@
  */
 import {Convert as convert} from '../../core/format';
 
-export enum HashType {
+export enum LockHashAlgorithm {
     Op_Sha3_256 = 0,
-    Op_Keccak_256 = 1,
-    Op_Hash_160 = 2,
-    Op_Hash_256 = 3,
+    Op_Hash_160 = 1,
+    Op_Hash_256 = 2,
 }
 
-export function HashTypeLengthValidator(hashType: HashType, input: string): boolean {
+/**
+ * Validate the hash length
+ * @param hashAlgorithm Secret lock hash algorithm
+ * @param input Hashed value
+ * @returns {boolean}
+ */
+export function LockHashAlgorithmLengthValidator(hashAlgorithm: LockHashAlgorithm, input: string): boolean {
```

Comment:
should we have a hash(hashAlgorithm: LockHashAlgorithm, input: string | Uint8Array):string that hashes the input using the right LockHashUtils (Hashers?). it's a similar switch statement

Reply:
It works differently from the Java. TS can do very limited in the enum class. Otherwise, have to export method like the validator. I think the static util class should be fine?

Reply:
maybe add the static function to LockHashUtils that receives a type and a value and does the proper switch statement